flow-nodes/api-builder-plugin-petstore/test.js

```diff
@@ -0,0 +1,74 @@
+const { expect } = require('chai');
+const { MockRuntime } = require('@axway/api-builder-sdk');
+
+const getPlugin = require('./index');
+
+function getLogger () {
```

Comment:
That would ideally be part of the work to make mockRuntime pass API builder props to getPlugin itself rather than making the user provide this information. I've got a draft for a ticket I need to create for this and I'll add to it.